src/store/getters.js

```diff
@@ -38,4 +39,8 @@ export default {
       ['ae_mainnet', 'ae_uat'].includes(activeNetwork.networkId) || process.env.RUNNING_IN_TESTS
     );
   },
+  explorerUrl: (_, { activeNetwork: { explorerUrl } }) => (hash) => {
```

Comment:
This is not just an getter, but getter-function, so I would name it
```suggestion
  getExplorerUrl: (_, { activeNetwork: { explorerUrl } }) => (hash) => {
```
or even `getExplorerPath`, because actually, it appends a specific path to explorer url